docs/modules/usage/how-to/custom-sandbox-guide.md

```diff
@@ -55,6 +61,28 @@ This can be an image you’ve already pulled or one you’ve built:
 sandbox_base_container_image="custom-image"
 ```
 
+### Additional Configuration Options
+
+The `config.toml` file supports several other options for customizing your sandbox:
+
+```toml
+[core]
+# Install additional dependencies at runtime build time
+# Can contain any valid shell commands
+# $OH_INTERPRETER_PATH is available as the path to the OH-specific Python interpreter
```

Comment:
Comments may need some clarification:
* First one says at runtime build time
* What is the comment about OH_INTERPRETER_PATH about?